src/installer/pkg/archives/dotnet-nethost.proj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Archives" Version="$(MicrosoftDotNetBuildTasksArchivesVersion)" />
+  </ItemGroup>
+
+
+  <Target Name="PublishToDisk">
+    <Error Condition="'$(OutputPath)' == ''" Text="Publishing to disk requires the OutputPath to be set to the root of the path to write to." />
+
+    <ItemGroup>
+      <FilesToPublish Include="$(DotNetHostBinDir)\$(LibPrefix)nethost$(LibSuffix)"
+            Destination="$(OutputPath)$(LibPrefix)nethost$(LibSuffix)" />
+      <FilesToPublish Include="$(DotNetHostBinDir)\$(LibPrefix)libnethost$(StaticLibSuffix)"
+            Destination="$(OutputPath)$(LibPrefix)libnethost$(StaticLibSuffix)" />
+      <FilesToPublish Condition="'$(TargetsWindows)' == 'true'"
+            Include="$(DotNetHostBinDir)\nethost.lib"
+            Destination="$(OutputPath)$(LibPrefix)nethost.lib" />
+      <FilesToPublish Include="$(DotNetHostBinDir)\nethost.h"
+            Destination="$(OutputPath)$(LibPrefix)nethost.h" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(FilesToPublish)"
+          DestinationFiles="%(FilesToPublish.Destination)" />
+  </Target>
+
+  <Target Name="PublishSymbolsToDisk">
+    <Error Condition="'$(SymbolsOutputPath)' == ''" Text="Publishing to disk requires the SymbolsOutputPath to be set to the root of the path to write to." />
+
+    <Copy SourceFiles="$(DotNetHostBinDir)\dotnet.pdb"
```

Comment:
We will also want the `nethost.pdb` so people can avoid suppressing linking warnings.

Reply:
I've updated the project to create the symbols archive and publish it in the same manner as the nethost archive.